io/src/main/java/io/smallrye/common/io/Messages.java

```diff
@@ -0,0 +1,23 @@
+package io.smallrye.common.io;
+
+import static java.lang.invoke.MethodHandles.lookup;
+
+import java.nio.file.FileSystem;
+import java.nio.file.Path;
+
+import org.jboss.logging.annotations.Message;
+import org.jboss.logging.annotations.MessageBundle;
+
+@MessageBundle(projectCode = "SRCOM", length = 5)
+interface Messages {
```

Comment:
Maybe also use it for logging?

```suggestion
interface Messages extends BasicLogger {
```